README.md

```diff
@@ -8,4 +13,17 @@ Mozilla Push server and Push Endpoint utilizing built with:
 - twisted
 - DynamoDB
 
-[Autopush Documentation on ReadThedocs](https://autopush.readthedocs.io/)
+[codecov-svg]: https://img.shields.io/codecov/c/github/mozilla-services/autopush/master.svg
+[codecov]: https://codecov.io/github/mozilla-services/autopush?branch=master
+[mpl-svg]: https://img.shields.io/badge/License-MPL%202.0-blue.svg
+[mpl]: https://opensource.org/licenses/MPL-2.0
+[travis-badge]: https://travis-ci.org/mozilla-services/autopush.svg?branch=master
+[travis]: https://travis-ci.org/mozilla-services/autopush
+[circleci-badge]: https://circleci.com/gh/mozilla-services/autopush.svg?style=shield&circle-token=074ae89011d1a7601378c41a4351e1e03f1e8177
+[circleci]: https://circleci.com/gh/mozilla-services/autopush
+[docs-badge]: https://readthedocs.org/projects/docs/badge/?version=latest
+[docs]: https://requires.io/github/mozilla-services/autopush/requirements/?branch=master
+[requiresio-badge]: https://requires.io/github/mozilla-services/autopush/requirements.svg?branch=master
```

Comment:
in your description, it says you're removing requires.io, but this looks like it's being added?